msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/AccountsSupplier.java

```diff
@@ -10,10 +10,10 @@
 
 class AccountsSupplier implements Supplier<Set<IAccount>> {
 
-    AbstractClientApplicationBase clientApplication;
+    AbstractApplicationBase clientApplication;
```

Comment:
I think the `AccountSupplier` can still work with the `AbstractClientApplicationBase`. Since this is not needed for managed identity, we do not need any account specific APIs in the base class.

Reply:
Since I also moved the getAccounts() method back into AbstractClientApplicationBase , this can definitely just use AbstractClientApplicationBase like before. Fixed in the latest commit.